modules/products/bazel/src/main/kotlin/com/metalbear/mirrord/products/bazel/BazelExecutionListener.kt

```diff
@@ -0,0 +1,154 @@
+package com.metalbear.mirrord.products.bazel
```

Comment:
Need someone on macOS to test this works with SIP.

Reply:
I was using the [tutorial Java project](https://github.com/bazelbuild/bazel/blob/master/site/en/start/java.md#get-the-sample-project)

Reply:
![image](https://github.com/user-attachments/assets/cd115122-0519-4340-8b00-36ff9039db38)
![image](https://github.com/user-attachments/assets/75432ab8-0962-4cdb-b824-3c724933a252)

:(